### PR TITLE
LPS-140077 Remove AUI from journal_template.jsp

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+			content: Liferay.Language.get('no-template'),
+			label: 'p',
+		});
+	};
+
+	const clearTemplate = delegate(
+		form,
+		'click',
+		`#${namespace}clearddmTemplateButton, #${namespace}ddmTemplateTypeDefault`,
+		onClearTemplate
```
Comment (p2kmgcl) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:142]: Maybe you can inline this function, it is not being used anywhere else.

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	};
+
+	const selectTemplate = delegate(
+		form,
+		'click',
+		`#${namespace}selectDDMTemplateButton`,
+		onSelectTemplate
+	);
+
+	eventDelegates.push(selectTemplate);
```
Comment (p2kmgcl) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:115]: I think you can just declare `eventDelegates` here, so you don't need to declare all these variables:

```js
const eventDelegates = [
  delegate(form, 'click', ...),
  delegate(another, 'event', ...),
  ...
];
```
```

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+	Liferay.Util.toggleRadio(
+		`${namespace}ddmTemplateTypeCustom`,
+		`${namespace}customDDMTemplateContainer`,
+		''
+	);
+
+	Liferay.Util.toggleRadio(
+		`${namespace}ddmTemplateTypeDefault`,
+		'',
```
Comment (p2kmgcl) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:155]: Just being curious but, what is `''` for?

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+					),
+					type: 'info',
+				});
+			},
+			selectEventName: eventName,
+			title: Liferay.Language.get('templates'),
+			url: createPortletURL(portletURL, {
+				eventName,
+				p_p_state: windowState,
+				...(ddmStructure != null && {ddmStructureId}),
```
Comment (p2kmgcl) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:103]: Triple equal:tm: `ddmStructure !== null`.
Also this is a bit hard to follow, maybe a short if would be easier to read. Doesn't it fail if `ddmStructure` is `null`?

File: modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js [hunk 5]
```diff
@@ -88,7 +88,7 @@ export default function ({
 					});
 
 				openToast({
-					container: form,
+					container: form.parentElement.previousElementSibling,
```
Comment (p2kmgcl) [inline modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:91]: :thinking: which element is this? Can't we use a selector?

Comment (veroglez) [reply modules/apps/journal/journal-content-web/src/main/resources/META-INF/resources/js/JournalTemplate.js:91]: This is because, this toast appeared under the form. Talking with design, it was decided to change it on top of the form. For this reason this element is being taken. I haven't used a selector because this element has no `id` and its `class` is very generic. It seemed like the best way to do it.
